modules/andes-core/broker/src/main/java/org/wso2/andes/server/txn/QpidDistributedTransaction.java

```diff
@@ -242,4 +242,12 @@ public void forget(Xid xid) throws UnknownDtxBranchException, IncorrectDtxStateE
     public void setTimeout(Xid xid, long timeout) throws UnknownDtxBranchException, AndesException {
         distributedTransaction.setTimeout(xid, timeout);
     }
+
+    /**
+     * Retrieve the {@link Xid}
+     * @return {@link Xid}
+     */
+    public Xid getXid() {
```

Comment:
I think we shouldn't add methods to classes for only logging purposes. That hurts the encapsulation.